dp-game-webapp/app/game/play/page.tsx

```diff
@@ -60,6 +63,23 @@ export default function Play() {
     ? savedVariance
     : adjustedVariance(conversionRate);
 
+  const shuffleQuestionOrder = () => {
+    const questionOrder: QuestionIndex[] = [];
+    for (let i = 0; i < NUM_QUESTIONS; i++) {
+      questionOrder.push({ index: i, noised: false });
+      questionOrder.push({ index: i, noised: true });
+    }
+
+    function shuffleArray(arr) {
```

Comment:
this is a pretty abstract function. is there a benefit for defining it inside the shuffleQuestionOrder instead of outside of it?